tOpt/opt_util.py

```diff
@@ -12,9 +12,9 @@
 
 
 class ConvergenceOpts(object):
-    def __init__(self, max_iter: int = 20, max_it_without_decrease: int = 5,
-                       convergence_e:    float = 1e-2,
-                       convergence_grms: float = 0.1,  convergence_gmax: float = 0.15,
+    def __init__(self, max_iter: int = 400, max_it_without_decrease: int = 40,
```

Comment:
We need to find a way to make this more configurable. I need to minimize the optimization cycles for my application and 0.2kcal/mol is not a big changes in my case.
I have committed changes that move the conf_opts creation to the constructor of BatchOptimizer. now we have to agree on how to set the default values. I can see multiple options:
- All conf_opts parameter could be set by command line options. This is a bit tedious.
- There could be  configuration file in the local or users home dir with the convergence criteria that is read for default values.